canonical/ubuntu.com
# Landing page redesign feedback

## Done

- Applied design changes as outlined on mock-up
- Drive by:
  - Removed unused class on /hpe
  - Ran djlint 
  - Upgraded to latest vanilla (v4.13)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13955.demos.haus/
- Check against [design](https://www.figma.com/design/Le4TTzyOFbcbqTPJ1tVoZT/U.com---Home-and-footer-(rebranding)?node-id=2158-5048&m=dev) and [doc](https://docs.google.com/document/d/1ySJxQbqVdeH4Tra0zwBm2Tn0s56kFGnEF7d8xDRTxwU/edit)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-11309
